app/views/pages/terms_conditions.html.erb

```diff
@@ -0,0 +1,181 @@
+      any third party.
+    </p>
+
+    <p class="govuk-body">
+      We reserve the right to restrict or deny you access to all or some part of the Site if in our opinion, you have
+      failed to comply with these General Terms.
+    </p>
+
+    <h3 class='govuk-heading-m'>
+      Hyperlinking to the Teachers: claim back your student loan repayments website
+    </h3>
+
+    <p class="govuk-body">
+      We do not object to you linking directly to pages on this site and you do not need to ask permission to do so.
+      However, we do not permit our pages to be loaded into frames on your site. The Teachers: claim back your student
+      loan repayments pages must be displayed in the user's entire browser window.
+    </p>
+
+    <h3 class='govuk-heading-m'>
+      Third Party Content and Hyperlinking from the Service
+    </h3>
+
+    <p class="govuk-body">
+      Where the Service contains links to other sites and resources, we are not liable or responsible for the content or
+      reliability of those websites and resources and do not necessarily endorse the views expressed within them.
+    </p>
+
+    <p class="govuk-body">
+      We aim to replace broken links to other sites but cannot guarantee that these links will always work as we have no
+      control over the availability of other sites.
+    </p>
+
+    <h3 class='govuk-heading-m'>
+      Virus protection
+    </h3>
+
+    <p class="govuk-body">
+      We make every effort to check and test material at all stages of production. It is always wise for you to run an
+      anti-virus program on all material downloaded from the Internet as we do not guarantee our service will be secure
+      or free from bugs or viruses. We cannot accept any responsibility for any loss, disruption or damage to your data
+      or your computer system which may occur whilst using material derived from this website.
+    </p>
+
+    <p class="govuk-body">
+      You must not misuse our service by knowingly introducing viruses, trojans, worms, logic bombs or other material
+      that is malicious or technologically harmful. You must not attempt to gain unauthorised access to our service,
+      the server on which our service is stored or any server, computer or database connected to our service. You must
+      not attack our site via a denial-of-service attack or a distributed denial-of service attack. Each of these acts
+      is a criminal offence. We will report any such offence to the relevant law enforcement authorities and co-operate
+      with them to determine your identity.
+    </p>
+
+    <h3 class='govuk-heading-m'>
+      Disclaimer and Liability
+    </h3>
+
+    <p class="govuk-body">
+      The content of this Service includes both content created by us (included but not limited to content that helps
+      you use the Service) and third-party content. We do not:
```

Comment:
I don't think we do include any 3rd party content directly.

Reply:
No I don't think so either, this was a bit of CYA legal speak. I'd like to leave it in for now and defer to someone who is a bit more of an expert in this field.

Reply:
I believe this was necessary for Teaching Vacancies, because that service displayed user entered data (such as job descriptions), which we don't have anything like. But yes, let's check with someone legal.